ts/components/session/settings/SessionSettings.tsx

```diff
@@ -589,6 +592,42 @@ export class SettingsView extends React.Component<SettingsViewProps, State> {
     ];
   }
 
+  private getBlockedUserSettings(): Array<LocalSettingType> {
+    const results: Array<LocalSettingType> = [];
+    const model = window.getConversations();
```

Comment:
we should use `BlockedNumberController.getBlockedNumbers()` here instead of relying on conversations.

The reason being is that conversations can be deleted but the number will still be blocked.